build.sbt

```diff
@@ -74,7 +74,7 @@ lazy val noop = crossProject(JSPlatform, JVMPlatform)
 
 lazy val slf4j = project
   .settings(commonSettings)
-  .dependsOn(core.js)
```

Comment:
Actually, that was discussed when @armanbilge was adding it https://github.com/typelevel/log4cats/pull/592#pullrequestreview-865833002.

Reply:
Oops. This is 💯  my mistake and I _really_ must not have been paying attention since your comment point it out 😳 I think I was trying to change `coreJVM` to `core.jvm` but I wrote `core.js` ... sorry everyone.

Reply:
In a Just world, either the build system or the compiler should have given you a fatal warning for this. This is a way too easy mistake to make.

Reply:
What really baffles me is that the tests run and pass using the JS artifact ...

Reply:
@armanbilge It's my fault too. I wasn't insistent and thought it's no matter what we'd use in that place :)